maestro-test/src/test/kotlin/maestro/test/RhinoJsEngineTest.kt

```diff
@@ -0,0 +1,58 @@
+package maestro.test
+
+import com.google.common.truth.Truth.assertThat
+import maestro.js.RhinoJsEngine
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mozilla.javascript.RhinoException
+
+class RhinoJsEngineTest : JsEngineTest() {
```

Comment:
These tests document known behaviors that are different from GraalJS. Common tests live in the base JsEngineTest class.